### ISSUE TITLE
Getting error message in logs

### ISSUE THREAD

**Issue by roberkane212** [dwaan/homebridge-adb#63]
Firestick- connect - connect -/bin/sh: adb: command not found 
Don’t know what this means

**Comment by roberkane212** [comment]
I have actually fixed this issue, however now all I get in homebridge logs is - Device disconnected? Trying to reconnect. I have done all steps, I have the right ip, and I also have the right config in my config.json. I’m not sure what I am doing wrong

**Comment by dwaan** [comment]
> I have actually fixed this issue, however now all I get in homebridge logs is - Device disconnected? Trying to reconnect. I have done all steps, I have the right ip, and I also have the right config in my config.json. I’m not sure what I am doing wrong 

From previous issues that I saw, Firestick tend to disconnected from network when it goes to sleep/powered off. Try running the plug-ins when your firestick is ON, and see if it still output the logs. If you saw that logs message, it means your installation work, it just that it can't connect to your device.

**Comment by roberkane212** [comment]
@dwaan  I’m not sure if anyone else has this issue anymore, but I fixed it sorry for not closing this. Once I connected to the firestick via usb. Then plugged it back into the computer I had to start homebridge then, in terminal again type Adb connect ipaddress:5555 and it was right in HomeKit ready to be used. And again you are right it constantly needs reconnection after it goes into sleep. Hopefully that can be fixed in the future by Amazon.
